pages/protected.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+// Non-empty subtitle to overwrite the sub navi of original page
+echo rex_view::title('YDeploy: Protected Page', ' ');
+
+echo rex_view::error('The page <code>'.rex_be_controller::getCurrentPage().'</code> is protected in deployed instances!');
```

Comment:
Vllt noch bissl mehr info, was das heißt und warum?

Ein user - selbst wenn er redaxo gut kennt - kann ggf. Mit der meldung in dieser form noch nix anfangen.
Er erkennt eine sperrung aber versteht nicht warum.

Reply:
Ja, kann ich noch ergänzen.
Wobei der User normalerweise ja gar nicht bis hier kommt, sondern sich dann eher vorher schon fragt, warum der die Page nicht mehr in der Navigation findet.

Reply:
Hm das stimmt. Ggf sollte die info eher dann im generellen backend layout irgendwie untergebracht sein (so das man es sieht aber nicht im weg steht)

Reply:
Log dich mal ins Backend von redaxo.org ein. Da siehst du, wie wir zurzeit die Live-Seite markieren (über yakme). Und auch wie das wirkt mit den ausgeblendeten Pages (aktuell auch über yakme).
Dieses Badge oben in der Mitte kommt ja eventuell auch noch direkt in ydeploy rein.

Reply:
(Dort ist es allerdings noch so, dass man die Pages über die direkte URL aufrufen kann, yakme blendet die nur per CSS aus)

Reply:
> Dort ist es allerdings noch so, dass man die Pages über die direkte URL aufrufen kann

Was ich durchaus schon gemacht hatte, um schnell mal eben ein Modul zu kopieren oder auch mal rein zu schauen. Ging  schneller als sich erst Lokal alles einzurichten.

Reply:
nur als randinfo .. bei redaxo.org werden templates und co deaktiviert und sind auch nicht über die URL erreichbar

Reply:
D.d. Vielleicht solle man die seiten readonly zulassen un nicht komplett sperren?

Reply:
Ursprünglich war mein Plan, die Pages zu schützen, sie aber in der neuen System/YDeploy-Page aufzulisten, und dort eine Möglichkeit zu schaffen, sie für die aktuelle Session freizugeben.
Aber dann dachte ich mir, eigentlich ist es nun mal schon richtig, dass sie geschützt sind, und ich spare mir das erstmal. 
@tbaddade Aber du fändest das also sinnvoll/praktisch?

Reply:
ich finde es schon gut, wenn man sie für die aktuelle Session freigeben könnte.. Es gibt immer so Notfälle und man hat seinen Rechner und die Installation nicht parat ..

Reply:
> Aber dann dachte ich mir, eigentlich ist es nun mal schon richtig, dass sie geschützt sind, und ich spare mir das erstmal.

Wenn man es richtig macht, sollten die Seiten gar nicht erreichbar sein.

> Aber du fändest das also sinnvoll/praktisch?

Aber wie @dergel schon schrieb, es gibt manchmal diese Fälle eben, wo es schon sinnvoll ist mal reinzuschauen. Ich denke, hier wären wir an dem Punkt, wo man das Adminrecht aufteilen sollte in Superadmin und Admin.

Reply:
Vllt kann der admin auf so einer seite auch sagen „notfall, ich weiß was ich tue“ und ist für seine session dann freigeschaltet?